scylladb/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/gofiber/storage/scylladb
+
+go 1.20
+
+require (
+	github.com/gocql/gocql v1.6.0
+	github.com/stretchr/testify v1.3.0
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/golang/snappy v0.0.3 // indirect
+	github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+)
+
+replace github.com/gocql/gocql => github.com/scylladb/gocql v1.11.1
```

Comment:
I dont know if dependabot will track this, may be better to fix the require import at the top of the go.mod